File: app/src/main/java/de/tum/in/tumcampusapp/models/tumcabe/RoomFinderSchedule.java
package de.tum.in.tumcampusapp.models.tumcabe;

public class RoomFinderSchedule {
    String start;
    String end;
    long event_id;
    String title;

    public RoomFinderSchedule(String start, String end, long event_id, String title) {
        this.start = start;
        this.end = end;
        this.event_id = event_id;
        this.title = title;
    }

    public String getStart() {
        return start;
    }

    public String getEnd() {
        return end;
    }

    public long getEvent_id() {
        return event_id;
    }

    public String getTitle() {
        return title;
    }
}

File: app/src/main/java/de/tum/in/tumcampusapp/component/tumui/roomfinder/model/RoomFinderSchedule.kt
package de.tum.`in`.tumcampusapp.component.tumui.roomfinder.model

@Deprecated(
"""should be removed
 Logic from WeekViewFragment and RoomFinderSchedule should be reimplemented
 in new NavigationDetailsFragment, instead of legacy RoomFinderDetailsActivity.
 More info: https://github.com/TUM-Dev/Campus-Android/pull/1462"
"""
)
data class RoomFinderSchedule(
    var start: String = "", // TODO: these should probably be DateTimes instead of Strings
    var end: String = "",
    var event_id: Long = -1,
    var title: String = ""
)
